rasa/core/processor.py

```diff
@@ -500,19 +500,22 @@ def _should_handle_message(tracker: DialogueStateTracker):
             or tracker.latest_message.intent.get("name") == USER_INTENT_RESTART
         )
 
+    def is_action_limit_reached(
+        self, num_predicted_actions: int, should_predict_another_action: bool
+    ) -> bool:
+        """Check whether the maximum number of predictions has been met"""
```

Comment:
```suggestion
        """Check whether the maximum number of predictions has been met.
       
        Args:
            num_predictes_actions: Number of predicted actions.
            should_predict_another_action: Whether the last executed action allows
            for more actions to be predicted or not.
            
        Returns:
            `True` if the limit of actions to predict has been reached.
        """
```

Reply:
Ah, sorry, was not sure how complete of docstrings I should put